app/src/main/res/layout/activity_greeting.xml

```diff
@@ -15,4 +15,15 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+            android:text="Log out"
```

Comment:
Layout looks good ! However, considering the Cirrus warnings and refactoring the texts could enhance with reusability of strings.